apps/web/src/components/Shared/Navbar/MobileDrawerMenu.tsx

```diff
@@ -0,0 +1,120 @@
+import { Button } from '@components/UI/Button';
+import { SwitchHorizontalIcon, XIcon } from '@heroicons/react/outline';
+import formatHandle from '@lib/formatHandle';
+import getAvatar from '@lib/getAvatar';
+import isGardener from '@lib/isGardener';
+import isStaff from '@lib/isStaff';
+import { Trans } from '@lingui/macro';
+import type { Profile } from 'lens';
+import Link from 'next/link';
+import type { FC } from 'react';
+import { useAppStore } from 'src/store/app';
+import { useGlobalModalStateStore } from 'src/store/modals';
+
+import Slug from '../Slug';
+import AppVersion from './NavItems/AppVersion';
+import Contact from './NavItems/Contact';
+import Logout from './NavItems/Logout';
+import Mod from './NavItems/Mod';
+import ReportBug from './NavItems/ReportBug';
+import Settings from './NavItems/Settings';
+import StaffMode from './NavItems/StaffMode';
+import Status from './NavItems/Status';
+import ThemeSwitch from './NavItems/ThemeSwitch';
+import YourProfile from './NavItems/YourProfile';
+
+const MobileDrawerMenu: FC = () => {
+  const profiles = useAppStore((state) => state.profiles);
+  const currentProfile = useAppStore((state) => state.currentProfile);
+  const setShowMobileDrawer = useGlobalModalStateStore((state) => state.setShowMobileDrawer);
+
+  const setShowProfileSwitchModal = useGlobalModalStateStore((state) => state.setShowProfileSwitchModal);
```

Comment:
```suggestion
  const setShowMobileDrawer = useGlobalModalStateStore((state) => state.setShowMobileDrawer);
  const setShowProfileSwitchModal = useGlobalModalStateStore((state) => state.setShowProfileSwitchModal);
```

removed new like 🙈  OCD hits me 🤣